raiden-dapp/src/App.vue

```diff
@@ -43,20 +43,27 @@ import ReceivingDisabledDialog from '@/components/dialogs/ReceivingDisabledDialo
     AppHeader,
     OfflineSnackbar,
     UpdateSnackbar,
-    ReceivingDisabledDialog,
+    NotificationArea,
   },
 })
 export default class App extends Mixins(NavigationMixin) {
   canReceive!: boolean;
-  showReceivingDisabled = false;
 
   destroyed() {
     this.$raiden.disconnect();
   }
 
   @Watch('canReceive')
   onCanReceiveChanged(canReceive: boolean | undefined) {
-    this.showReceivingDisabled = canReceive === false;
+    if (canReceive !== false) {
+      return;
+    }
+    this.$store.dispatch('notifications/notify', {
+      title: this.$t('receiving-disabled-dialog.title'),
+      description: this.$t('receiving-disabled-dialog.body'),
+      importance: NotificationImportance.HIGH,
+      context: NotificationContext.WARNING,
```

Comment:
Also, maybe this should have `duration: -1`, so the user must explicitly dismiss?